db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+lifehacks = Lifehack.create(title: 'How to tie a tie', description: 'tie a knot')
```

Comment:
Useless assignment to variable - `lifehacks`. Did you mean `lifehacks_2`?<br>Line is too long. [81/80]